src/verifact_agents/claim_detector.py

```
@@ -1,97 +1,382 @@
+        # Remove common noise patterns
+        text = re.sub(r'\b(um|uh|er|ah)\b', '', text, flags=re.IGNORECASE)
+
+        # Normalize common abbreviations
+        text = re.sub(r'\bvs\.', 'versus', text, flags=re.IGNORECASE)
+        text = re.sub(r'\betc\.', 'etcetera', text, flags=re.IGNORECASE)
+
+        # Final whitespace cleanup after all substitutions
+        text = " ".join(text.split())
+        return text
+
+    def _deduplicate_claims(self, claims: List[Claim]) -> List[Claim]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)